dotcom-rendering/src/components/SignInGateSelector.importable.tsx

```diff
@@ -706,6 +706,25 @@ const SignInGateSelectorAuxia = ({
 				);
 				if (data !== undefined) {
 					setAuxiaGateDisplayData(data);
+
+					const treatmentId =
+						data.auxiaData.userTreatment?.treatmentId;
+					if (treatmentId) {
+						// Record the fact that Auxia has returned a treatment. This is not a VIEW event, so we use the RETURN action here
+						await submitComponentEventTracking(
+							{
+								component: {
+									componentType: 'SIGN_IN_GATE',
+									id: treatmentId,
+								},
+								action: 'RETURN',
```

Comment:
this is otherwise the same as the VIEW event further down - https://github.com/guardian/dotcom-rendering/pull/13564/files#diff-cc933475d587696c53144cb65852989ce22bf2b1fa13b10bdee3268f8966963bR822